docs/UserGuide.adoc

```diff
@@ -130,17 +243,238 @@ Examples:
 
 * `list` +
 `delete 2` +
-Deletes the 2nd person in the address book.
+Deletes the 2nd person in the list of people.
 * `find Betsy` +
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+=== Deleting a policy : `deletepolicy`
+
+Deletes the specified policy from the address book. +
+Format: `deletepolicy INDEX`
+
+****
+* Deletes the policy at the specified `INDEX`.
+* The index refers to the index number shown in the displayed policy list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `listpolicies` +
+`delete 2 +
+Deletes the 2nd policy in the list of policies.
+* `find senior` +
+`delete 1` +
+Deletes the 1st policy in the results of the `find` command.
+
+=== Deleting policies from a person : `unassignpolicy`
+
+Removes a policy from the person at the specified index. +
+Format: `unassignpolicy INDEX pol/POLICY [MORE_POLICIES]`
+
+****
+* Removes the policies to the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `listpeople` +
+`unassignpolicy 2 pol/Senior Care` +
+Removes the `Senior Care` policy from the 2nd person in the list of people.
+* `find Betsy` +
+`unassignpolicy 1 pol/Health Insurance` +
+Removes the `Health Insurance` policy from the 1st person in the results of the `find` command.
+
+=== Deleting tags from a person : `deletetag`
+
+Deletes a tag from the person at the specified index. +
+Format: `deletetag INDEX t/TAG [MORE_TAGS]`
+
+****
+* Deletes the tags from the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* Any number of tags can be deleted.
+****
+
+Examples:
+
+* `listpeople` +
+`deletetag 2 t/high_priority` +
+Deletes a `high_priority` tag from the 2nd person in the list of people.
+* `find Betsy` +
+`deletetag 1 t/high_risk` +
+Deletes a `high_risk` tag from the 1st person in the results of the `find` command.
+
+=== Deleting tags from a policy : `deletepolicytag`
+
+Deletes a tag from the policy at the specified index. +
+Format: `deletepolicytag INDEX t/TAG [MORE_TAGS]`
+
+****
+* Deletes the tags from the policy at the specified `INDEX`.
+* The index refers to the index number shown in the displayed policies list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* Any number of tags can be deleted.
+****
+
+Examples:
+
+* `listpolicies` +
+`deletepolicytag 2 t/high_priority` +
+Deletes a `high_priority` tag from the 2nd policy in the list of policies.
+* `findpolicy Senior` +
+`deletepolicytag 1 t/high_risk` +
+Deletes a `high_risk` tag from the 1st policy in the results of the `find` command.
+
 // end::delete[]
 === Clearing all entries : `clear`
 
 Clears all entries from the address book. +
 Format: `clear`
 
+=== Incorrect NRIC Alert
```

Comment:
I was thinking we can split the features up. What do you think?
```
3.1 Basic Features
3.1.1 Help ...
3.2 Input validation
3.2.1 Incorrect NRIC alert
```

Reply:
Actually I don't think we should do that? I think some of the features are like intertwined with each other like the search functions are a bit related to the basic commands like for example to add tags or policies to people (basic command) we would need to find them first (search function) then it would make sense to put search before adding tags. Does that make sense?

Reply:
Yup search example makes sense. 
My concern was that the features will be too long for the user; how about we stick to this for now, and abstract the features out later if the problem (of features being too long to be readable) surfaces?

Reply:
oh yup okay!!